src/Components/Server/src/BlazorPack/SequenceOfT.cs

```diff
@@ -1,5 +1,5 @@
-// Copyright (c) Andrew Arnott. All rights reserved.
-// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Comment:
Not sure this is a legitimate change. Files under `/BlazorPack` are being used on a shared-source basis. I don't think we get to change the copyright header.